docs/data-tunnel/route-types/01-tss.md

```diff
@@ -1,13 +1,81 @@
 
-During this period, you may encounter intermittent updates, changes, and limited availability as we work to enhance its features and performance.
+## Creating a TSS Tunnel
 
-We appreciate your patience and understanding as we aim to deliver a robust and improved testing experience. Stay tuned for more updates and release information!
+### Deploying a Target Contract
 
-:::
+A target contract must implement the `IDataConsumer` interface to support the `BandTunnelRouter` contract. The implementation details are provided below:
 
-> This route type is currently only available on BandChain v3 testnet.
+```solidity
+interface IDataConsumer {
+    /**
+     * @dev Processes the relayed message.
+     *
+     * The relayed message must be evaluated from the tunnelRouter contract and
+     * verified by the tssVerifier contract before forwarding to the target contract.
+     *
+     * @param data The decoded tss message that is relayed from the tunnelRouter contract.
+     */
+    function process(PacketDecoder.TssMessage memory data) external;
+
+    /**
+     * @dev Activates the tunnel and set the sequence on tunnelRouter contract.
+     *
+     * This function deposits tokens into the vault and sets the latest sequence on the
+     * tunnelRouter contract if the current deposit in the vault contract exceeds a threshold.
+     * The transaction is reverted if the threshold is not met.
+     *
+     * This function should be called by the contract owner.
+     *
+     * @param latestSeq The new sequence of the tunnel.
+     */
+    function activate(uint64 latestSeq) external payable;
+
+    /**
+     * @dev Returns The tunnelRouter contract address.
+     */
+    function tunnelRouter() external view returns (address);
+
+    /**
+     * @dev Returns The tunnelId of the contract address.
+     */
+    function tunnelId() external view returns (uint64);
+}
+```
+
+### Creating a TSS Tunnel in BandChain
+
+To create a TSS Tunnel using the command-line interface (CLI), execute the following command:
+
+```bash
+bandd tx tunnel create-tunnel tss [destination-chain-id] [destination-contract-address] [encoder] [initial-deposit] [interval] [signalDeviations-json-file]
+```
+
+Parameters:
+
+- **destination-chain-id**: The ID of the destination chain supported by Band Protocol. A list of supported chains will be publicly provided soon.
+- **destination-contract-address**: The contract address on the destination chain that implements the `IDataConsumer` interface to support the `BandTunnelRouter` contract.
+- **encoder**: The type of data and the method of encoding for that data. Currently, we support FIXED_POINT_ABI (1) and TICK_ABI (2).
```

Comment:
explain more a bit about each encoder type. i'm not sure if we have some place explain about this.